dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,331 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.fixture.StatementExecution;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.DdlStatementExecution;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.Options;
+import dbfit.util.TypeNormaliserFactory;
+import fit.TypeAdapter;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment {
+
+    @Override
+    public void afterConnectionEstablished() throws SQLException {
+        if (currentConnection.getMetaData().supportsTransactions()) {
```

Comment:
After doing some testing with setting `autocommit` it seems that this doesn't affect that `getMetaData().supportsTransactions()` returns. It's always `false` in Informix mode and `true` in `ANSI` mode.

It seems like we don't need this method override at all in fact (at least with the acceptance tests that we have, including of course `CommitAndRollback`).

I think we can just leave the autocommit state be the default.